### PR TITLE
Add Window Door Sensor documentation for LIVISI integration

### PR DESCRIPTION
**Pull request by StefanIacobLivisi**
## Proposed change
Add documentation for LIVISI Window Door Sensor

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/82659
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/livisi.markdown [hunk 1]
```diff
@@ -2,14 +2,16 @@
 title: LIVISI SmartHome
 description: Access and control your LIVISI Smart Home Controller (SHC) and its connected RWE/innogy devices.
 ha_category:
+  - BinarySensor
   - Switch
 ha_iot_class: Local Push
-ha_release: 2022.12.0
+ha_release: 2023.01.0
```
Comment (frenck) [inline source/_integrations/livisi.markdown]: This should not change, as this is the original release the integration was added (not last updated).

File: source/_integrations/livisi.markdown [hunk 2]
```diff
@@ -2,14 +2,16 @@
 title: LIVISI SmartHome
 description: Access and control your LIVISI Smart Home Controller (SHC) and its connected RWE/innogy devices.
 ha_category:
+  - BinarySensor
```
Comment (frenck) [inline source/_integrations/livisi.markdown]: This category does not exist, I guess you meant `Binary Sensor` (with a space).

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_integrations/livisi.markdown (hunk 1 above)
Comment (StefanIacobLivisi) [reply source/_integrations/livisi.markdown]: I changed this, thank you.

File: source/_integrations/livisi.markdown (hunk 2 above)
Comment (StefanIacobLivisi) [reply source/_integrations/livisi.markdown]: I changed this, thank you.

**Review by frenck — DISMISSED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.

**Comment by planbnet** [comment]
@StefanIacobLivisi Please update the reference to the parent PR to https://github.com/home-assistant/core/pull/90220 
The original PR was closed and I have created a new one.

**Review by frenck — APPROVED** [review]

**Pull request merged**